### ISSUE TITLE
App crash every time I open the same folder.

### ISSUE THREAD

**Issue by tarasius** [files-community/Files#3548]
**Describe the bug**
Open one folder always triggers immediate crash with no error message shown to the user.

**To Reproduce**
Open folder D:\Docs folder with PDF, DOCX, PNG files and few subfolders. App closes.

**Expected behavior**
Open folder. If error occurs, show it to the user instead of silent crash.

**Desktop (please complete the following information):**
Windows 10 Enterprise, build 18363
Ver 1.2.2.0 from Microsoft Store.

**Log file**
```
2021-02-12 11:09:01.9419|INFO|FilesFullTrust.QuickLook|QuickLook detected: False|
2021-02-12 11:09:01.9414|INFO|FilesFullTrust.QuickLook|QuickLook detected: False|
2021-02-12 11:09:02.9179|INFO|FilesFullTrust.Program|Argument: GetOneDriveAccounts|
2021-02-12 11:09:03.3734|INFO|FilesFullTrust.Program|Argument: GetIconOverlay|
...
2021-02-12 11:09:05.7401|INFO|FilesFullTrust.Program|Argument: GetIconOverlay|
2021-02-12 11:10:23.0501|INFO|FilesFullTrust.Program|Argument: Terminate|
```

**Comment by gave92** [comment]
Hopefully the same as #3504

**Comment by jamestansx** [comment]
> Hopefully the same as #3504.
> 
> * @tarasius could you check if the crash happens also if you put the current folder in Grid View before going to that folder?
> * Could you check if the crash happens also if you enable the "open items with one click" option in Files settings?
> * Do get a notification saying "something went wrong" when the crash happens?

I was having the same issue too.  And by trying these:

- Changing to Grid View does solve this problem.
- Opening items with one click also works.
- There's no error notification when the crash happens

**Comment by tarasius** [comment]
@gave92
> * @tarasius could you check if the crash happens also if you put the current folder in Grid View before going to that folder?

It does not crash, but switches back from grid view back to list view.

> * Could you check if the crash happens also if you enable the "open items with one click" option in Files settings?

Yes, it crashes navigating to the same folder with this setting turned on.

> * Do get a notification saying "something went wrong" when the crash happens?

No. Silently closes after hanging for a few settings.

**Comment by gave92** [comment]
This issue should be resolved as of version 1.2.3 which is going to ship in a few days. If after the update you keep experiencing this issue please let us know!
